### PR TITLE
MP-9366 - Generate Custom Integration Event

### PR DESCRIPTION
**Pull request by dejobratic**

### CODE REVIEW

File: tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs [hunk 1]
```diff
@@ -7,15 +8,15 @@
 
 namespace Nox.Generator.Tests.Application;
 
-public class ApplicationEventTests: IClassFixture<GeneratorFixture>
+public class ApplicationEventTests : IClassFixture<GeneratorFixture>
```
Comment (gjorgji-grgovski-authoritypartners) [inline tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs]:
```suggestion
public class IntegrationEventTests : IClassFixture<GeneratorFixture>
```

File: tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs [hunk 2]
```diff
@@ -7,15 +8,15 @@
...
 {
     private readonly GeneratorFixture _fixture;
 
     public ApplicationEventTests(GeneratorFixture fixture)
```
Comment (gjorgji-grgovski-authoritypartners) [inline tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs]:
```suggestion
    public IntegrationEventTests(GeneratorFixture fixture)
```

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

File: tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs (hunk 1 above)
Comment (dejobratic) [reply tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs]: Changes were applied. Thanks for the suggestion.

File: tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs (hunk 2 above)
Comment (dejobratic) [reply tests/Nox.Generator.Tests/Application/ApplicationEventTests.cs]: Changes were applied. Thanks for the suggestion.

**Review by rochar — APPROVED** [review]
looks good

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Pull request merged**